tests/test_registry.py

```diff
@@ -261,19 +530,17 @@ def push():
         )
         public_key, private_key = Configuration.key_pair(key_pair_setting)
         result = registration.push(
```

Comment:
Can't this be `result = push()` like the above since the function does this same thing?